LightWeightSignalBus.cs

```diff
@@ -20,7 +18,7 @@ public static IDisposable Subscribe(Action<T> action)
 	public static void Unsusbscribe(Action<T> action) => SignalBus<T>.action -= action;
 
 	//Subscription registered for Lightweight version
-	public class Subscription : IDisposable
+	public class SignalSubscription : IDisposable
```

Comment:
I would keep as just "Subscription"
since its a nested class it would make sense to call it as that

the reasons wich is not called "SignalSubscription" are 2  
its not a signal subscription, in fact its a subscription to the signal bus of a specific signal type.
so it would have more sense to be called as "SignalBusSubscription" because as said is a subscription to the bus, not to the signal itself, and SignalBus<T>.SignalBusSubscription or SignalBus<T>.SignalSubscription would make it a little bit redundant in the naming